packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -350,3 +393,17 @@ export default class Sayt extends LitElement {
     `;
   }
 }
+
+/**
+ * The type of the callback expected to be passed to getDebounce.
+ */
+export interface requestSaytType extends debounce {
```

Comment:
In general, interfaces should start with a capital letter. That said, I don't think we'll need this interface in the new approach.